### PR TITLE
Tune k8s

### PR DESCRIPTION
**Pull request by webrouse**
Slack: https://keboola.slack.com/archives/CL11GKH6H/p1654776464313709

### CODE REVIEW

File: provisioning/kubernetes/templates/templates-api.yaml [hunk 1]
```diff
@@ -28,11 +28,9 @@ spec:
           image: $TEMPLATES_API_REPOSITORY:$TEMPLATES_API_IMAGE_TAG
           resources:
             requests:
-              cpu: "100m"
```
Comment (kacurez) [inline provisioning/kubernetes/templates/templates-api.yaml:31]: ten request na cpu by som tam nechal, podla toho scheduler efektivnejsie pripaduje pody na nody.

Comment (kacurez) [reply provisioning/kubernetes/templates/templates-api.yaml:31]: daj 200m pls

**Review by kacurez — APPROVED** [review]
lgtm

**Comment by webrouse** [comment]
(merged in: https://github.com/keboola/keboola-as-code/pull/715)

**Pull request closed without merging**